BerkeleyHCI/PolymorphicBlocks
# Cleanup

- Resolves #232, removes not_connected which wasn't really used anywhere else
- Resolves #219, renames all the example folders to the board name
  - Remove `Test` from the board names
  - Update .gitignore, should be a bit cleaner now
- Allow example designs that aren't meant to be board (all the blinky examples) to not generate netlists and stuff
- Remove gerbers, in the repo probably wasn't the right place to put them
- Add another parsing rule for JLC diodes
